package.json

```
@@ -3,7 +3,7 @@
   "version": "0.9.4",
   "scripts": {
     "build": "rm -rf .next && next build",
-    "dev": "next dev",
+    "dev": "NODE_OPTIONS='--max-old-space-size=8192' next dev",
```

Please do not modify shared scripts unless absolutely necessary.

```suggestion
    "dev": "next dev",
```